app/webpacker/styles/breakpoints.scss

```diff
@@ -3,7 +3,7 @@ $mq-breakpoints: (
   narrow: 380px,
   mobile: 500px,
   tablet: 768px,
-  desktop: 1000px,
+  desktop: 1100px,
```

Comment:
I'd suggest bumping this up a few more pixels to `1114px` so that the titles don't wrap over three lines between 1110px-1113px